### PR TITLE
chore: prepare 1.0.0-beta.1

### PR DESCRIPTION
**Pull request by saw-jan**
Prepare release for `1.0.0-beta.1`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,3 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 
 ## [Unreleased]
+
+### Added
+
+-   GherLint CLI tool
+-   First lint rule `indentation`
+-   Add `--fix` cli option to fix lint errors (https://github.com/gherlint/gherlint/pull/31)
+-   Add new lint rule `no_repetitive_step_keyword` (https://github.com/gherlint/gherlint/pull/52)
+-   Report invalid gherkin file as lint error (https://github.com/gherlint/gherlint/pull/63)
```
[the remark this answers is not in the document]
Comment (grgprarup) [reply CHANGELOG.md:16]: IMO, mentioning the date and version would be better.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,3 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Added
+
+-   GherLint CLI tool
```
Comment (grgprarup) [inline CHANGELOG.md:12]:
```suggestion
-   GherLint CLI tool
```
Is this a feature or description of the tool?

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,3 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+-   First lint rule `indentation`
+-   Add `--fix` cli option to fix lint errors (https://github.com/gherlint/gherlint/pull/31)
```
Comment (grgprarup) [inline CHANGELOG.md:14]:
```suggestion
-   Add `--fix` cli option to fix lint errors (https://github.com/gherlint/gherlint/pull/31)
```
Do we really need a repetitive `Add` keyword at the beginning, as all of these go under the `Added` heading?

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (grgprarup) [reply CHANGELOG.md:16]: Also, add changes in reverse order. New changes at the top.

[the remark this answers is not in the document]
Comment (saw-jan) [reply CHANGELOG.md:16]: when we release major version we will add version and date. For now it will be in beta

**Pull request merged**